.github/workflows/windows-acceptance-basic.yml

```diff
@@ -0,0 +1,101 @@
+# This workflow runs the command test suite on GitHub-based Windows Server runners.
+name: Windows Basic Acceptance
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+env:
+  GO_VERSION: '1.19'
+
+  # NOTE: some tests require that the version string returned
+  # by the server be SemVer-compatible
+  CONTAINERD_VERSION: '1.6.8'
+
+jobs:
+  acceptance-basic:
+    name: Windows Basic Acceptance
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 50
+    env:
+      GOTEST: gotestsum --
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [windows-2019, windows-2022]
+
+    defaults:
+      run:
+        shell: bash
```

Comment:
Should this be bash or PowerShell?

Reply:
Good catch, the initial version of the workflow contained way more steps and mostly used bash in them but none do any more so I'll be updating the workflow to have PS as the default.

Reply:
Updated the PR to use PowerShell as the default shell FWIW.